sdk/inc/azure/core/az_result.h

```diff
@@ -102,11 +102,7 @@ typedef enum
   // Platform
   AZ_ERROR_OUT_OF_MEMORY = _az_RESULT_MAKE_ERROR(
       _az_FACILITY_PLATFORM,
```

Comment:
We should remove this facility for now and move this `az_result` to core.

However, since that is internal implementation detail, we can decide and fix this in a separate PR, post-preview 4.